pressbooks/pressbooks-book-directory-fe
# Improve timestamp consistency by using UTC

This fixes #296 

This PR improves timestamp consistency by using UTC time. This will work better for searching books and will not cause problems with different timezones.

This PR also removes the [date-fns](https://github.com/date-fns/date-fns) library in favor of the lighter [dayjs](https://github.com/iamkun/dayjs) one.

#### How to test

1. Run `npm install` to install new dependencies
1. Apply last update filters and check if the filter works properly
1. Check if timestamps are generated as UTC
1. Run e2e tests
